test/test_slots.py

```diff
@@ -264,6 +264,79 @@ def test_nb_ops_power(self):
         tmp **= 42
         assert tmp == (p, 'inplace_power', 42, None)
 
+    def test_buffer(self):
+        import pytest
+        import sys
+        mod = self.make_module("""
+            typedef struct {
+                HPyObject_HEAD
+                int exports;
+            } FakeArrayObject;
+
+            static char static_mem[12] = {0,1,2,3,4,5,6,7,8,9,10,11};
+            static HPy_ssize_t _shape[1] = {12};
+            static HPy_ssize_t _strides[1] = {1};
+            static HPy_buffer fakebuf = {
```

Comment:
If `HPy_buffers` are meant to be short lived objects, should we perhaps fill in the buffer in the test, instead of reusing this static one?